testproject/django_file_form_example/templates/example_form.html

```diff
@@ -25,6 +26,11 @@
             <label for="{{ form.title.id_for_label }}">Title</label>
             {{ form.title }}
         </div>
+
+        {% if direct_to_s3 %}
```

Comment:
Is it correct that the upload method is set in this example project? I would expect it to be set in the `django-file-form` app.

Would it be possible to put the upload method in a `data` attribute, instead of an input? Perhaps it can be put on `dff-container`, like `data-files`. See `upload_widget.htm`.
The reason that I ask this is that an input could cause a conflict; somebody could add an `uploadMethod` input to a form.

Reply:
I agree that `example_form.html` is the wrong place for this piece of code since this requires users to add the HTML code to their own forms. `upload_widget.htm` is a much better place but as I commented below, I do not have a good way to pass such as information to this widget.

Reply:
This is now done by adding a hidden field.

Reply:
I see that the `s3_upload_dir` is now added in `django_file_form`. That's good.